### ISSUE TITLE
Issues while installing

### ISSUE THREAD

**Issue by shaunakjoshi12** [torchvideo/torchvideo#71]
Hi! The below 2 errors come up when installing from source

1. ERROR: Could not find a version that satisfies the requirement gulpio>=540.66 (from torchvideo) (from versions: none)

**Comment by CastielHuang** [comment]
Yes, I have the same question. I can't install either because of the error:" Could not find suitable distribution for Requirement.parse('gulpio>=540.66') ". I wanna know the solution. Can you contect with the blogger?

**Comment by SophieOstmeier** [comment]
I have the same issue. gulpio was moved to gulpio2, however, the requirements of this repo say gulpio. Could someone update the requirements please?
